PureBasicIDE/EditHistory.pb

```diff
@@ -378,6 +377,7 @@ Procedure History_FlushEvents()
       
     Wend
     
+    SetGadgetState(#GADGET_HistoryShutdown_Progress, 0)
```

Comment:
This too feels more like a bug in the ProgressbarGadget command and should be fixed there and not in the IDE.